src/main/java/com/conveyal/datatools/manager/auth/Auth0Connection.java

```diff
@@ -15,22 +16,38 @@
 
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.PublicKey;
+import java.security.SignatureException;
+import java.security.spec.InvalidKeySpecException;
 import java.util.Arrays;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 import static com.conveyal.datatools.common.utils.SparkUtils.haltWithMessage;
 import static com.conveyal.datatools.manager.DataManager.getConfigPropertyAsText;
+import static com.conveyal.datatools.manager.DataManager.hasConfigProperty;
 import static spark.Spark.halt;
 
 /**
+ * This handles verifying the Auth0 token passed in the Auth header of Spark HTTP requests.
+ *
  * Created by demory on 3/22/16.
  */
 
 public class Auth0Connection {
+    public static final String APP_METADATA = "app_metadata";
+    public static final String USER_METADATA = "user_metadata";
+    public static final String SCOPE = "http://datatools";
```

Comment:
Perhaps this string should be obtained from a config file

Reply:
Yea, perhaps you're right. Thoughts on a field name? `CUSTOM_AUTH0_CLAIM_NAMESPACE`?

Reply:
Actually, I'm not sure this is really necessary.  It shouldn't have to change between different instances like the other Auth0 config variables.  All that matters is that the value matches the namespace in the rule that is set up for the tenant.  To reduce setup overhead, I say we just keep this an unconfigurable constant.

Reply:
I'm don't have a very strong opinion on this, but I still lean towards having this in a config file because this scope is a variable that can be changed outside of this server application and if we only provide this hardcoded variable, a user would have to edit code.  It also doesn't have to be extra setup overhead if we include this variable as a default in an example config file.

Reply:
I'm just going to keep this hardcoded. The likelihood that anyone would actually need to change this value is very small and adding another configuration value that is only used for new Auth0 clients adds unneeded complexity here (even if it's just perceived complexity).